lib/middleware.js

```diff
@@ -161,6 +166,7 @@ var middlewares = module.exports = {
       'Cmd': [],
       'Dns': null,
       'Image': null,
+      'Labels': keypath.get(req, 'body.Labels') || {},
```

Comment:
When creating a container, "Labels" is a root level key on the body -- but when inspecting a container docker returns "Labels" under "Config" -- making labels a 2nd level child of the response